app/services/job-state.js

```diff
@@ -1,37 +1,37 @@
 import Service, { inject as service } from '@ember/service';
 import { task } from 'ember-concurrency';
+import { computed } from '@ember/object';
 import { filter, reads, gt } from '@ember/object/computed';
 
-const FINISHED_STATES = ['failed', 'canceled', 'passed'];
-const RUNNING_STATES = ['started'];
-const QUEUED_STATES = ['created', 'queued', 'booting', 'received'];
-const RUNNING_AND_FINISHED_STATES = RUNNING_STATES.concat(FINISHED_STATES);
-const DISPLAY_TIMEOUT_SEC = 30;
+export const FINISHED_STATES = ['failed', 'canceled', 'passed'];
+export const RUNNING_STATES = ['started'];
+export const QUEUED_STATES = ['created', 'queued', 'booting', 'received'];
+export const UNFINISHED_STATES = QUEUED_STATES.concat(RUNNING_STATES);
 
 export default Service.extend({
   store: service(),
 
-  jobs: reads('fetchJobs.lastSuccessful.value'),
-  jobsLoaded: gt('fetchJobs.performCount', 0),
+  jobs: reads('peekJobs.lastSuccessful.value'),
+  jobsLoaded: gt('fetchUnfinishedJobs.performCount', 0),
+  sortedJobs: computed('jobs', function () {
```

Comment:
It should probably have `jobs.@each.number` for proper tracking, shouldn't it? Or you may also use [sort](https://api.emberjs.com/ember/3.12/functions/@ember%2Fobject%2Fcomputed/sort) computed macro, although it's a bit awkward in use :)

Reply:
Oh yeah, good point. I put in `jobs.@each.number`, but yeah `sort` computed macro is another good option for sure